README.md

```diff
@@ -4,14 +4,62 @@ Library for managing [WebMock][] fixtures.
 
 [WebMock]: https://github.com/bblimke/webmock
 
-## Installation
+[Documentation](http://www.rubydoc.info/github/underdogio/webmock-fixtures).
```

Comment:
Yeah, rubydoc.info will auto-generate the docs from us.

We get the README + the in code docs.

For gems, it will generate docs for each version that we publish (rather than being just `master`).